### ISSUE TITLE
Finish HTML support

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/d43-catalog#35]
The html support of OBS is very brittle.
There are supposed to be some updates coming for the html printing pipeline that should fix this.

**Comment by neutrinog** [comment]
We need to get some updates to the html printing out before we can finish this.

**Comment by neutrinog** [comment]
duplicate of https://github.com/unfoldingWord-dev/d43-catalog/issues/37
